docs/entity_air_pollutants.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Air Pollutants Entity
+sidebar_label: Air Pollutants
+---
+
+## Properties
+
+> Properties should always only return information from memory and not do I/O (like network requests). Implement `update()` or `async_update()` to fetch data.
+
+| Name | Type | Default | Description
+| ---- | ---- | ------- | -----------
+| state | string | **Required** | The current PM 2.5 level.
+| particulate_matter_2_5 | float | **Required** | The particulate matter 2.5 (<= 2.5 μm) level.
+| particulate_matter_10 | float | **Required** | The particulate matter 10 (<= 10 μm) level.
+| temperature | float | `None` | The current temperature in °C or °F.
+| humidity | float | `None` | The current humidity in %.
+| air_quality_index | float | `None` | The Air Quality Index (AQI).
+| ozone | float | `None` | The O3 (ozone) level.
+| carbon_monoxide | float | `None` | The CO (carbon monoxide) level.
+| carbon_dioxide | float | `None` | The CO2 (carbon dioxide) level.
+| particulate_matter | float | `None` | The general particulate matter level.
```

Comment:
Are all these properties really necessary? Just because we can access some piece of data doens't mean we have to standardize it. I think that we should focus on the bare minimum of what people expect from air quality and implement that. Then look how it is used and go from there.

Reply:
At the moment it's somehow the cross section over the existing sensors.

Let me remove 1 or 2. :wink: